src/scheduled-query/azext_scheduled_query/_help.py

```diff
@@ -16,12 +16,6 @@
 type: command
 short-summary: Create a scheduled query.
 parameters:
-  - name: --action -a
-    short-summary: Add an action group and optional webhook properties to fire when the alert is triggered.
-    long-summary: |
-        Usage:   --action ACTION_GROUP_NAME_OR_ID [KEY=VAL [KEY=VAL ...]]
-
-        Multiple action groups can be specified by using more than one `--action` argument.
```

Comment:
Please add help messages for action_groups and custom_properties.

Reply:
Added help information in `_params.py`, do I still need to add it here?

Reply:
Yes. Please add help message for `custom_properties` with Usage